.github/workflows/test-all.yml

```diff
@@ -250,6 +276,32 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - uses: octokit/request-action@v2.x
+        id: get-jobs
+        with:
+          route: GET /repos/${{ github.repository }}/actions/runs/${{ github.run_id}}/jobs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we really have to do this in each job? Can't we do this once, in the final job that generates the status report? I.e. grab the complete jobs data once then, and extract all job ids in one script? Might even handle that from within python then?

Reply:
Yes, I can try to do this. Should save us some minutes of run time, since this download (~1s) adds up a bit.

Reply:
I mean if we do this, we could also skip the step with uploading the test-status artifact for each package, since all the info is available in this json from the REST API request.

Should I remove the artifact step and use the json instead to determine the job status for each package?

Reply:
Ohhhh, I see. OK, why not, sounds much simpler

Reply:
Of course then we must also make sure to extract the package name from the job name. But we need that anyway, so..

Reply:
I would then iterate over the metadata JSON files (instead of the test-status JSON files) and get the package names from there via `normalize_pkg_name`.